src/vumi2/routers.py

```diff
@@ -73,6 +74,7 @@ async def _get_matched_mapping_names(self, addr):
         for name, pattern in self.mappings:
             if pattern.match(addr):
                 matched_names.append(name)
```

Comment:
I'm planning on changing the way this function works, since it only needs to return first match but have it this way at the moment so that I can easily toggle between one and multiple matches while figuring out the test.